src/borg/archive.py

```diff
@@ -1569,13 +1569,17 @@ def process_dev(self, *, tarinfo, status, type):
 
     def process_symlink(self, *, tarinfo, status, type):
         with self.create_helper(tarinfo, status, type) as (item, status):
-            item.target = tarinfo.linkname
+            normalized_path = os.path.normpath(tarinfo.linkname)
+            safe_path = make_path_safe(normalized_path)
+            item.target = safe_path
```

Comment:
Hmm, after some more thoughts about this, I believe this would not be correct.

The symlink target (tarinfo.linkname) has to be seen like the contents of a user file and we must not change/edit that.

If a symlink points to `../../foo`, that must stay that way.

It's also not a safety issue, because extracting a symlink with such a target does no harm.

Reply:
I reverted that change.